src/components/externalLinks/ExternalLinksCRUD/ConfigureLinkAction.tsx

```diff
@@ -0,0 +1,185 @@
+                    <input
+                        placeholder="Enter name"
+                        value={link.name}
+                        onChange={(e) => onNameChange(index, e.target.value)}
+                    />
+                    {link.invalidName && getErrorLabel('name')}
+                </div>
+                <div className="link-clusters mr-12">
+                    <span>Clusters*</span>
+                    <ReactSelect
+                        placeholder="Select clusters"
+                        name={`link-clusters-${index}`}
+                        value={selectedClusters}
+                        options={clusters}
+                        formatOptionLabel={formatOptionLabelClusters}
+                        onChange={(selected) => onClusterSelection(index, selected)}
+                        isMulti={true}
+                        hideSelectedOptions={false}
+                        closeMenuOnSelect={false}
+                        inputValue={clusterSearchInput}
+                        onBlur={() => {
+                            setClusterSearchInput('')
+                        }}
+                        onInputChange={(value: string, actionMeta: InputActionMeta) => {
+                            if (actionMeta.action === 'input-change') {
+                                setClusterSearchInput(value)
+                            }
+                        }}
+                        components={{
+                            IndicatorSeparator: null,
+                            ClearIndicator: null,
+                            ValueContainer,
+                            Option,
+                        }}
+                        styles={{
+                            ...multiSelectStyles,
+                            ...customMultiSelectStyles,
+                            menuList: (base, state) => ({
+                                ...customMultiSelectStyles.menuList(base, state),
+                                maxHeight: '210px',
+                            }),
+                            container: (base, state) => ({
+                                ...base,
+                                marginTop: '6px',
+                            }),
+                            control: (base, state) => ({
+                                ...customMultiSelectStyles.control(base, state),
+                                width: '278px',
+                            }),
+                            placeholder: (base) => ({
+                                ...base,
+                                color: 'var(--N500)',
+                            }),
+                        }}
+                    />
+                    {link.invalidClusters && getErrorLabel('clusters')}
+                </div>
+                {showDelete && (
+                    <div className="link-delete mt-24 cursor">
+                        <Delete className="icon-dim-20" onClick={() => deleteLinkData(index)} />
```

Comment:
same here

Reply:
As discussed, this component is going to be updated in the next feature branch PR in which all these changes will be covered.